react-components/src/components/Button/Button.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+export interface ButtonProps {
+  label: string;
+}
+
+const Button = (props: ButtonProps) => {
+  return <button>{props.label}</button>;
+};
+
+export default Button;
```

Comment:
Do we want to have a default export or just normal one? `cogs` and `data-exploration-components` use standard export as we do in Reveal.